### PR TITLE
KVGarg.md: Add EWA project work report

### PR DESCRIPTION
**Pull request by KVGarg**

### CODE REVIEW

**Comment by KVGarg** [comment]
@sks444 @margobra8 @li-boxuan Can you provide some review on the report?

And, the line limit is exceeding, for which I've already added a change in [one of my commits in another PR](https://github.com/coala/projects/pull/731/files#diff-e7d7d93b4c8d0df9c9bf21a7d47c83b5).

File: _reports/KVGarg.md [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+phase:
+  - Bonding: https://gitlab.com/coala/GSoC/gsoc-2019/milestones/14
+  - Phase 1: https://gitlab.com/coala/GSoC/gsoc-2019/milestones/15
+  - Phase 2: https://gitlab.com/coala/GSoC/gsoc-2019/milestones/16
+  - Phase 3: https://gitlab.com/coala/GSoC/gsoc-2019/milestones/18
+bio: >
+  I am a final year student (expected graduation month: June 2020) studying
+  Computer Science and Engineering at Chandigarh College of Engineering and
+  Technology, Chandigarh. I participated in GSoC'19 with
+  [coala](http://coala.io) open-source organization.
```
Comment (li-boxuan) [inline _reports/KVGarg.md]: Use https

**Comment by margobra8** [comment]
LGTM other than Travis failing

**Comment by KVGarg** [comment]
Fixing up that! Need to add my report to `ignores`

**Pull request merged**